phpBB/adm/style/acp_avatar_options_upload.html

```diff
@@ -17,52 +17,52 @@
 	<div class="avatar-cropper-buttons" id="avatar-cropper-buttons">
 		<div class="button-group">
 			<button class="button" type="button" title="{{ lang('ZOOM_IN') }}" data-cropper-action="zoom,0.1">
-				<i class="icon fa-search-plus fa-fw"></i>
+				{{ Icon('font', 'search-plus', '', true, 'fas') }}
```

Comment:
These will actually not require specifying fas since that's the default, see also the ucp file